test/extensions/filters/http/jwt_authn/filter_integration_test.cc

```diff
@@ -14,9 +20,52 @@ namespace HttpFilters {
 namespace JwtAuthn {
 namespace {
 
-std::string getFilterConfig(bool use_local_jwks) {
+const std::string HeaderToFilterStateFilterName =
```

Comment:
Can you use `CONSTRUCT_ON_FIRST_USE` or the singleton pattern or `const char[]` here?

Reply:
Done